stdlib/http/src/main/java/org/ballerinalang/net/http/compiler/websocket/WebSocketClientResourceValidator.java

```diff
@@ -38,8 +38,9 @@ void validateOnOpenResource() {
     }
 
     void validateEndpointParameter() {
-        if (paramDetails != null && !paramDetails.isEmpty() && !WebSocketConstants.FULL_WEBSOCKET_CLIENT_NAME.equals(
-                paramDetails.get(0).type.toString())) {
+        if (paramDetails != null && !paramDetails.isEmpty() && !(WebSocketConstants.FULL_WEBSOCKET_CLIENT_NAME.equals(
+                paramDetails.get(0).type.toString()) || WebSocketConstants.FULL_FAILOVER_WEBSOCKET_CLIENT_NAME.equals(
+                paramDetails.get(0).type.toString()))) {
```

Comment:
Shall we add another class named `WebSocketFailoverClientResourceValidator` or such to validate the failover service? Let's follow the class structure.